### PR TITLE
Fix the example of the code to get pipelines in README file

### PR DESCRIPTION
**Pull request by funkycode**
Here is working example.

### CODE REVIEW

**Comment by funkycode** [comment]
As from discussion (https://github.com/codefresh-io/go-sdk/issues/1) it seems like some functions were renamed. this example works perfectly for 0.5.0, probably should be adjusted according to changes in 0.5.1

File: README.md [hunk 1]
```diff
@@ -9,19 +9,33 @@
...
-    cf.GetPipelines()
+    options, err := utils.ReadAuthContext(path, "")
+    if err != nil {
+        fmt.Println("Failed to read codefresh config file")
+        panic(err)
+    }
+    clientOptions := codefresh.ClientOptions{Host: options.URL,
+        Auth: codefresh.AuthOptions{Token: options.Token}}
+    cf := codefresh.New(&clientOptions)
+    pipelines, err := cf.GetPipelines()
```
[the remark this answers is not in the document]
Comment (funkycode) [reply README.md]: Fixed that. Checked - it is working.

**Comment by olegsu** [comment]
Have to agree with you on it https://github.com/codefresh-io/go-sdk/pull/2#issuecomment-453843598
I would use it right away

**Comment by funkycode** [comment]
I'll try to push another merge request tomorrow to use context instead of options (or maybe options to have both optionally?)

**Comment by olegsu** [comment]
In general, users probably need the ability to create a client from `$HOME/.cfconfig` and from options.
So I guess both will work.
Let's take it separately tomorrow?
Appreciate your contribution 👍

**Pull request merged**